text-ui-test/EXPECTED.TXT

```diff
@@ -1,7 +1,13 @@
 Hello from
- ____        _        
-|  _ \ _   _| | _____ 
-| | | | | | | |/ / _ \
-| |_| | |_| |   <  __/
-|____/ \__,_|_|\_\___|
+ _       _ 
+| |  _  | |_   _ ____ ___
+| | | | | | |_/ |  _ \  _ \ 
+| |_| |_| |\___ |    <  __/
+\___/\___/ \____|_| \_\___|
+
+
+^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^
+Hi! I'm Wyre, your Personal Assistant Chatbot! :>
+What can I do for you today?
+^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^-^
```

Comment:
This isn't really a comment on code style, but there seems to be a lack of tests implemented for your PR. Maybe a few tests to check your program could be implemented.

Reply:
I definitely need to work on that, thank you for reminding me!